src/app/views/datasets-new/edit-metadata/DatasetMetadataController.jsx

```diff
@@ -176,7 +176,14 @@ export class DatasetMetadataController extends Component {
                 isGettingVersionMetadata: false, 
                 versionIsInCollection: mappedVersion.collection, 
                 instanceID: mappedVersion.instanceID, 
-                versionIsPublished: mappedVersion.versionIsPublished});
+                versionIsPublished: mappedVersion.versionIsPublished
+            });
+
+            // if version state is edition-confirmed load in dimensions  
+            // labels and descriptions from last published version
+            if (version.state === "edition-confirmed") {
```

Comment:
Picky comment but best practice would be to have the constant as the first arg in the comparison as it's safer if version is nil.
```
if ("edition-confirmed" === version.state) { ... } 
```